django/db/models/expressions.py

```diff
@@ -565,6 +568,7 @@ def as_sqlite(self, compiler, connection, **extra_context):
         return sql, params
 
 
+@deconstructible(path='django.db.models.TemporalSubtraction')
```

Comment:
There is no need to simplify paths for private APIs that we will never use in migrations. Please revert changes to the `TemporalSubtraction`, `DurationExpression`, `CombinedExpression`, `Expression`, `SQLiteNumericMixin`, `ResolvedOuterRef`, `ExpressionList`, `Col`, `Ref`, etc.

Reply:
Are `Expression` and `ExpressionList` private APIs too?

Reply:
`ExpressionList` is a private API, you will not find it in docs. `Expression` is not private, but it's not usable on its own, you need to create a subclass.